### ISSUE TITLE
Multiple modular encounters

### ISSUE THREAD

**Issue by funkymonkeymonk** [funkymonkeymonk/tts-marvel-champions-automated#12]
I'd like to support adding multiple modular encounters to the encounter deck.

**Comment by SwedeRacer** [comment]
Yes! This is something I was going to ask for. Here's what I was thinking (if possible; I'm not familiar with the capabilities of Tabletop Simulator or Lua):

Instead of a dropdown for encounter set selection, there could be a list of checkboxes. When you select a scenario, the default encounter set(s) would be automatically selected. So, for instance, if you choose Rhino, then Bomb Scare will be pre-checked. But then you can uncheck that set and check a different one, or check multiples, or whatever, before clicking the button to build.
